core/iwasm/fast-jit/fe/jit_emit_numberic.c

```diff
@@ -367,6 +419,61 @@ compile_int_div(JitCompContext *cc, IntArithmetic arith_op, bool is_i32,
                                                 right, res);
             }
         }
+
+        if (jit_reg_is_const(left)) {
```

Comment:
This code piece can not be executed as the code above has handled all possible branches. How about moving the code into `compile_int_div_no_check`, it doesn't need to check exception again as possible exception checks have been done before calling it.